### ISSUE TITLE
Feature request: Add documentation for adding SSO to a bot

### ISSUE THREAD

**Issue by stephanbisser** [pnp/generator-teams#315]
### 💡 Idea

Add a tutorial to describe how to add SSO to a bot with all the necessary steps (AAD config, code changes, ...) to make it easier for folks to build a SSO enabled bot

### Is your feature related to a bug

no

**Comment by andrewconnell** [comment]
What about just pointing to existing docs? The MS Learning module for Teams SSO includes [explination about SSO + bots](https://docs.microsoft.com/training/modules/msteams-sso/6-bots-sso) and the [HOL exercise](https://docs.microsoft.com/training/modules/msteams-sso/7-exercise-bots-sso) uses Yo Teams (latest) to walk the user through this.

**Comment by stephanbisser** [comment]
@andrewconnell yes that would be great! I was just wondering if in the future those docs will be replaced by Teams Toolkit based docs so we need to offer an alternative for yo teams devs.

**Comment by RickVanRousselt** [comment]
Indeed great suggestion, let's link it in the docs for now, and keep track should it change.

**Comment by andrewconnell** [comment]
@stephanbisser said:

> I was just wondering if in the future those docs will be replaced by Teams Toolkit based docs so we need to offer an alternative for yo teams devs.

I know that's their desire, but I work with the MS Learn team to refresh all the MSTeams modules in the associate learning path quarterly... at this point, I've heard no plans to port them over to the toolkit... so you're at least good for a few months.

**Comment by andrewconnell** [comment]
The other benefit is that there's a published screencast on the M365Dev YouTube channel for each module, including SSO. This is the 7th video in the playlist for the SSO module... it + all remaining ones cover what you'd have to create in terms of docs: https://www.youtube.com/watch?v=OVFy_sJkNU4&list=PLWZJrkeLOrbZ3uG8Xb8yOUeWu7UDu4Q_-&index=7

**Comment by stephanbisser** [comment]
Perfect, thanks @andrewconnell for the information on this one! Let's use that for now
